### PR TITLE
Adding cnn_bridge to documentation index for distro

### PR DESCRIPTION
**Pull request by wew84**

### CODE REVIEW

File: kinetic/distribution.yaml [hunk 1]
```diff
@@ -1317,6 +1317,21 @@ repositories:
...
     status: maintained
+  cnn_bridge:
+    doc:
+      type: git
+      url: https://github.com/wew84/cnn_bridge.git
+      version: master
+    release:
+      tags:
+        release: release/kinetic/{package}/{version}
+      url: https://github.com/wew84/cnn_bridge.git
```
Comment (tfoote) [inline kinetic/distribution.yaml]: This is not a release repository. Did you intend to make a binary release? If so please follow: http://wiki.ros.org/bloom/Tutorials/FirstTimeRelease

File: melodic/distribution.yaml [hunk 2]
```diff
@@ -630,6 +630,21 @@ repositories:
...
     status: maintained
+  cnn_bridge:
+    doc:
+      type: git
+      url: https://github.com/wew84/cnn_bridge.git
+      version: master
+    release:
+      tags:
+        release: release/melodic/{package}/{version}
+      url: https://github.com/wew84/cnn_bridge.git
```
Comment (tfoote) [inline melodic/distribution.yaml]: This is also not a release repository. Note that if you use bloom then it will generate the PRs for you.

**Review by tfoote — CHANGES_REQUESTED** [review]
Please see the link about making a release. If you only want to be indexed please just remove the release stanzas.

**Comment by wew84** [comment]
Created a proper release repo for release will rerelease

**Pull request closed without merging**